alpha-blog/app/views/articles/_article.html.erb

```diff
@@ -4,10 +4,15 @@
       <div class="col-8 mt-4">
         <div class="card text-center shadow mb-5 bg-white rounded">
           <div class="card-header font-italic">
-            by <%= link_to article&.user&.username, user_path(article.user) %>
+            by <%= link_to article&.user&.username, user_path(article.user), class: "no-underline" %>
```

Comment:
1. if you do `link_to nil`, will it work? If no, your code of using `&` will raise exception anyway right?
2. why one side do `article&.user` and another do `article.user` ?